index.html

```diff
@@ -167,7 +178,9 @@
         <div class="col-md-7">
             <div :id="'phylogeny-' + phylogenyIndex" class="panel panel-info" v-for="(phylogeny, phylogenyIndex) of testcase.phylogenies">
                 <div class="panel-heading">
-                    <a href="#" class="close glyphicon glyphicon-remove" @click="confirm('Are you sure you want to delete this phylogeny?', () => testcase.phylogenies.splice(phylogenyIndex, 1))"></a>
+                    <!-- Minimize button for panel -->
+                    <a href="#" class="close glyphicon glyphicon-collapse-up" onclick="$(this).parent().parent().find('.panel-body, .panel-footer').toggle(300); $(this).toggleClass('glyphicon-collapse-up glyphicon-collapse-down')"></a>
```

Comment:
For my taste, that's getting long as a piece of code that is not subjected to any testing nor linting. Consider moving to a function?

BTW there's a plugin for eslint that allows it to be used for code in HTML pages.  I lost track over whether we've enabled that here yet.